demo/index.css

```diff
@@ -63,8 +63,9 @@ amp-video {
   margin-bottom: 10px;
 }
 
-.dropdown-wrapper .snippet-container {
-  margin-top: 125px;
```

Comment:
This style isn't used and I think it's supposed to be used with [this](https://github.com/torch2424/aesthetic-css/blob/master/demo/elements/dropdown.html#L22). But it will introduce a new class in the example, so I update it.

Reply:
Ahhh! Thank you very much for catching this! :smile: Yeah I made this such a long time ago, I probably had a small hiccup haha :joy: